applications/FluidDynamicsApplication/custom_processes/apply_compressible_navier_stokes_boundary_conditions_process.cpp

```diff
@@ -29,12 +29,12 @@ ApplyCompressibleNavierStokesBoundaryConditionsProcess::ApplyCompressibleNavierS
     mSubsonicBCs.reserve(Parameters["subsonic_boundary_conditions"].size());
     mSupersonicBCs.reserve(Parameters["supersonic_boundary_conditions"].size());
 
-    for(auto var_settings : Parameters["subsonic_boundary_conditions"])
+    for(const auto& var_settings : Parameters["subsonic_boundary_conditions"])
```

Comment:
i am really not sure of the behaviour of Parameters if you get a reference to it. I would leave out this ones